PWGDQ/Core/MCSignalLibrary.h

```diff
@@ -317,9 +346,14 @@ MCSignal* o2::aod::dqmcsignals::GetMCSignal(const char* name)
     signal = new MCSignal(name, "mumu pairs from j/psi decays", {prong, prong}, {1, 1}); // signal at pair level
     return signal;
   }
-  if (!nameStr.compare("eeFromPsi2S")) {
+  if (!nameStr.compare("eeFromPsi2s")) {
```

Comment:
Same comment as above. I suggest to use capital S

Reply:
Thanks for comments. Should I use capital S for all of Psi2s Cases?